uktrade/data-hub-frontend
# Amend design of client contact details in Investment form

## Description of change

This PR adds a `<ContactInformation>` component to the CRM and adds it to the Investment form.  The component is just a paragraph with a link in, and a Details component with some information around what is needed to make a contact, however, this same design is to be implemented on the Interaction form too, so I thought it would be better placed as a reusable component. 


## Test instructions

Go to the investments tab and click add investment project. Search for a company and add an investment type, and click continue. When you get to the contacts field, you should see the change demonstrated in the screenshots below. The add a new contact link should redirect you to the contacts form, and send you back when you've added the new contact. 

## Screenshots
### Before

<img width="1071" alt="Screenshot 2022-03-14 at 11 55 16" src="https://user-images.githubusercontent.com/46787754/158167069-959409d6-1c81-4b3c-8041-a4db9b95300f.png">


### After

<img width="1032" alt="Screenshot 2022-03-14 at 11 51 24" src="https://user-images.githubusercontent.com/46787754/158166859-0e5da211-673d-4bec-9774-eebeb40178e9.png">


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
